views/cube/cube_list.pug

```diff
@@ -164,6 +165,162 @@ block cube_content
   .container#cubelistarea
   #react-root
 
+  // Filter Modal
```

Comment:
It doesn't block this PR, but I would love if we could start trying to get new UI elements implemented in React instead of Pug. Especially for mostly-client-side features.

Reply:
I haven't spent enough time with react to do this, but moving forward I will. I just wanted to get this done for the release tonight.